### DISCUSSION TITLE
Display train accuracy

### DISCUSSION THREAD

**Discussion by caiocvsilva** [open-mmlab/mmaction2#2782] [Q&A]
Hey,
 
I am trying to check the train accuracy of my model, not the validation accuracy. I noticed that before the migration to the new version of the code, it seems that the code displayed the train accuracy (top1 and top5) in the log, but right now it only shows other metrics:

`mmengine - INFO - Epoch(train)     [1][156/156]  lr: 3.9872e-04  eta: 62 days, 20:27:52  time: 0.6706  data_time: 0.0042  memory: 6222  loss: 2.8455  loss_cls: 2.8455`

I saw some other discussions that hinted on that direction, but the solutions were flags that do not exist in the code after the migration.

Any help on getting the accuracy is welcome.

**Comment by YuHoChau** [comment]
Same issue here. Have you solved the problem? Thanks.
